### PR TITLE
Improve cli doc layout

### PR DESCRIPTION
**Pull request by benolayinka**
#### Summary

Improve CLI doc layout. @htdvisser I ended up doing this using GenMDDoc instead of the shortcodes, because I would like to use a separate page for each command anyway, and there is no way to generate pages from json in hugo -> https://discourse.gohugo.io/t/automatically-generate-pages-from-json-data-file/13494

This only generates ttn-lw-stack docs, though. I couldn't get tti-lw-stack docs to generate, and that will require a more manual flow to run the generator in the private repo anyway.

EDIT: updated to build cli docs from `tti-lorawan-stack` repository

#### Screenshots

Bildschirmfoto 2021-09-30 um 14 41 59

#### Notes for Reviewers

...

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Review by nejraselimovic — COMMENTED** [review]
Great content.

This looks a bit weird, the title is too small and text is not in line with other docs pages. Any way you can fix this to look like on the rest of the docs?

**Comment by htdvisser** [comment]
> This only generates ttn-lw-stack docs, though. I couldn't get tti-lw-stack docs to generate, and that will require a more manual flow to run the generator in the private repo anyway.

This is going to complicate maintaining this, since we used to generate everything (also API docs) from the TTI repository. Building CLI docs from the TTN repository will mean that we double the work.

File: cmd-gen.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+git clone https://github.com/TheThingsNetwork/lorawan-stack.git lorawan-stack
+
+cd lorawan-stack
+
+make init
```
Comment (htdvisser) [inline cmd-gen.sh:5]: This will take a long time, and I don't think it's actually necessary to just generate some docs

**Comment by benolayinka** [comment]
> > This only generates ttn-lw-stack docs, though. I couldn't get tti-lw-stack docs to generate, and that will require a more manual flow to run the generator in the private repo anyway.
> 
> This is going to complicate maintaining this, since we used to generate everything (also API docs) from the TTI repository. Building CLI docs from the TTN repository will mean that we double the work.

Alright, generating from TTI docs is the right way to do it anyway. I'll update this so we manually copy out from TTI, rather than cloning from TTN within this repo

**Comment by benolayinka** [comment]
> Great content.
> 
> This looks a bit weird, the title is too small and text is not in line with other docs pages. Any way you can fix this to look like on the rest of the docs? 

Which title and text?

File: cmd-gen.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+go run ./cmd/ttn-lw-cli gen-md-doc -o ../doc/content/ttn-lw-cli
+
+# Command End Device Templates has two parents and no doc is generated, prune
+sed -i.bak 's/end-devices_templates/templates/g' ../doc/content/ttn-lw-cli/ttn-lw-cli_end-devices.md
```
Comment (htdvisser) [inline cmd-gen.sh:9]: `sed -i` works differently on macOS/Linux/Windows, so can we try to do this differently?

**Comment by nejraselimovic** [comment]
> > Great content.
> > This looks a bit weird, the title is too small and text is not in line with other docs pages. Any way you can fix this to look like on the rest of the docs? 
> 
> Which title and text?

"Command-Line Interface" title and the text next to it. The title is bigger on other docs and that text usually goes below the title.

**Review by htdvisser — APPROVED** [review]
Nothing to add

**Pull request merged**